background.js

```diff
@@ -296,6 +380,21 @@ function handleTeamscaleResponse(request, action) {
 	persistAndBroadcastSingleEvent(event);
 }
 
+function handleError(request){
+	const actionString = '🛑 Connection Refused';
+	const text = 'Connecting to ' + configOptions[serverOptionId] + 'failed. Is the server running?';
```

Comment:
I intentionally formulated this generic enough so that it matches also for the Teamscale server case. Theoretically, it is possible that Teamscale instances are not running. WDYT? 
To make it a bit more clear, I rephrased the second sentence as follows: "Is the server running and reachable?"

Reply:
👍

Please add the missing space before "failed"

Reply:
Sure, done 👍